study.md

```diff
@@ -19,38 +19,55 @@ Please write the code that would take the following integer `007` and return the
 string `"700"`
 
 ```ruby
-  # your answer here
+
+007.to_s.rjust(3, '0').reverse
+=> "700"
+
 ```
 
 Please write the code that takes the following array `[23,56,3,7]` sorts it
 then reverses that array returning and modifying the refrence to the original
 array.
 
 ```ruby
-  # your answer here
+arr = [23,56,3,7]
+=> [23,56,3,7]
+arr.sort!
+=> [3, 7, 23, 56]
+array.reverse
+=> [56, 23, 7, 3]
```

Comment:
Original array is modified, and reverse method is applied to the modified (original, not copy) array. So, reversed array is the modified one.